lib/core/defaultPageCompleteCheck.js

```diff
@@ -2,9 +2,9 @@
 
 module.exports = `
 return (function(waitTime) {
-    try { 
-        const end = window.performance.timing.loadEventEnd;
-        return (end > 0) && (Date.now() > end + waitTime);
+    try {
+        var end = window.performance.timing.loadEventEnd;
+        return end > 0;
```

Comment:
We skip the use of the extra waitTime, any reason for that? That's an way for the user to make the test test longer without using the more complicated pageCompleteCheck.

Reply:
The reason we removed it here is because when we playback a recorded webpage (through mitmproxy) on Chrome, this code hangs. We moved this wait to here - I can change it back or we can try to find another solution to this: https://github.com/sitespeedio/browsertime/pull/1028/files/88bd5c79e52e2fb4b1ec2263bb1510666fd41ca2#diff-598e2038200e98260b612f9a4fc397d7R178

Reply:
Ah I see, we do the same for WebPageReplay since the date object is static: https://github.com/sitespeedio/browsertime/blob/master/docker/scripts/start.sh#L91 

Push this as is and the I can change the code so we use performance.now instead (or do you also override that?). Since we plan to release this as a major, it would be nice to fix so it works out of the box both and without a reply proxy, that will make things simpler.

Reply:
Sounds good, and I don't think we override performance.now anywhere.